### ISSUE TITLE
Recursive folder deletion

### ISSUE THREAD

**Issue by stephan281094** [frozeman/meteor-build-client#13]
#### The problem

When executing `meteor-build-client ../` in de Meteor application folder. It deletes **everything**. Looking at the code, this makes sense. But when you want to export the static files one folder higher, it's unexpected behavior and can even cause users to lose all their work.
#### Steps to reproduce
- Create a folder called `foo/` and `cd` into it.
- `meteor create app`
- `cd app`
- `meteor-build-client ../`
- You get the following error:
  
  ``` bash
  Bundling Meteor app...
  
  fs.js:612
    return binding.rmdir(pathModule._makeLong(path));
                   ^
  Error: ENOENT, no such file or directory './../'
      at Object.fs.rmdirSync (fs.js:612:18)
      at deleteFolderRecursive (/usr/local/lib/node_modules/meteor-build-client/meteor.js:48:12)
      at Object.module.exports.build (/usr/local/lib/node_modules/meteor-build-client/meteor.js:56:9)
      at /usr/local/lib/node_modules/meteor-build-client/main.js:68:20
      at null._onTimeout (/usr/local/lib/node_modules/meteor-build-client/queue.js:41:11)
      at Timer.listOnTimeout [as ontimeout] (timers.js:110:15)
  ```
  
  And the `foo/` folder is completely empty.

In this scenario the user would lose all files inside `foo/`, including the folder where the Meteor application is located. This should be fixed.

**Comment by lukeadams** [comment]
Two things I think would fix it:
 - We should warn when deleting if the directory already exists and isn't empty (whoops, deleted ~/). Can override with --force/-f for build systems or scripts
 - Check if `buildDir` is part of the folder tree for the current meteor app (`path.resolve('./')`)

**Comment by frozeman** [comment]
I think we should prevent using the parent folder all together, as its quite weird to put you build files in the parent folder, where it also contains the source folder then.

**Comment by stephan281094** [comment]
I'm closing this due to inactivity.

**Comment by matikucharski** [comment]
I think this should be reopened and it should be added warning that not empty dir is going ot be deleted. We can add flag to skip the warning but I think this could save some ppls from accidentally running the script in /home folder ;)

**Comment by dr-dimitru** [comment]
Reopen, thanks to @matikucharski

**Comment by dr-dimitru** [comment]
@matikucharski @stephan281094 solved in [`v1.3.0`](https://github.com/frozeman/meteor-build-client/releases/tag/1.3.0)

Feel free to reopen it in case if the issue is still persists on your end.
